packages/legend-application-query/src/components/QuerySetup.tsx

```diff
@@ -276,77 +277,99 @@ export const QuerySetupLandingPage = withQuerySetupLandingPageStore(
         applicationStore.config.studioUrl,
       );
     const showAllActionGroup = (): void => setupStore.setShowAllGroups(true);
-
+    const toggleAssistant = (): void =>
+      applicationStore.assistantService.toggleAssistant();
     useEffect(() => {
       setupStore.initialize(showAdvancedActions, showAllGroups, tagToFocus);
     }, [setupStore, showAdvancedActions, showAllGroups, tagToFocus]);
 
     return (
```

Comment:
I have to warn you one small thing here is that there are children screens of query setup (i.e. when you click on one of the modes), so we probably missed the status bar there

Reply:
status bar was added in the BasicQuerySetup, which will make sure there will be one status bar in every single children screen. (also double-checked by rendering each child component one by one) :) 
<img width="532" alt="Screenshot 2022-11-29 at 11 57 55 AM" src="https://user-images.githubusercontent.com/73408381/204594969-867d1221-9c2b-4082-ba23-223f4113c3e6.png">